src/Serialization/containers.jl

```diff
@@ -11,10 +11,20 @@ function type_params(obj::S) where {T, S <:MatVecType{T}}
   end
   
   params = type_params.(obj)
-  @req params_all_equal(params) "Not all params of Vector or Matrix entries are the same, consider using a Tuple for serialization"
+  @req params_all_equal(params) "Not all params of Array entries are the same, consider using a Tuple for serialization"
```

Comment:
I don't understand this change. The function here does not take `Array`, but just vectors, matrices and SRwos